app/controllers/admin/api/api_docs_services_controller.rb

```diff
@@ -133,15 +132,23 @@ def destroy
 
   def api_docs_params(*extra_params)
     permit_params = %i[name body description published skip_swagger_validations service_id] + extra_params
-    params.require(:api_docs_service).permit(*permit_params)
+    permitted_params = params.require(:api_docs_service).permit(*permit_params)
+
+    service_id = permitted_params[:service_id]
+    if service_id.present?
+      permitted_params[:owner_id] = service_id
```

Comment:
I think this part could be handled in the model, e.g.
https://github.com/3scale/porta/blob/1792c848d20564505f89ee214ba560a5c84fb362/app/models/metric.rb#L10
https://github.com/3scale/porta/blob/1792c848d20564505f89ee214ba560a5c84fb362/app/models/metric.rb#L244-L252